<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetSet</class>
 <widget class="QWidget" name="NetSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>592</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>11</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8" stretch="1,1,1,1,1,1,2">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>电脑钥匙设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1,2,1,1,0">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>串口选择</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="keyComSelCombo"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>串口设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="keyComSetCombo"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>串口超时(秒)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="keyComTimeOut"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>模拟盘设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1,2,1,1,0">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>串口选择</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mnpComSelCombo"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>串口设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mnpComSetCombo"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>串口超时(秒)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mnpComTimeOut"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>优特五防设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,1,2,1,1,0">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>串口选择</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ytComSelCombo"/>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>串口设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ytComSetCombo"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>串口超时(秒)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ytComTimeOut"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>模拟屏2设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2,1,2,1,1,0">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>串口选择</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mnp2ComSelCombo"/>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>串口设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mnp2ComSetCombo"/>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>串口超时(秒)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mnp2ComTimeOut"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>闭锁器设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2,1,2,1,1,0">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>串口选择</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="bsqComSelCombo"/>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>串口设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="bsqComSetCombo"/>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>串口超时(秒)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="bsqComTimeOut"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>五防机网络</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>网络A网IP：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="netAEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>网络B网IP：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="netBEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2,0">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string> 网络端口：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="portEdit"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="title">
        <string>五防机与电脑钥匙</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7" stretch="2,2">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>五防机编号：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="wfNumberEdit"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
